adafruit_ble_cycling_speed_and_cadence.py

```diff
@@ -75,10 +80,10 @@ class _CSCMeasurement(ComplexCharacteristic):
 
     uuid = StandardUUID(0x2A5B)
 
-    def __init__(self):
+    def __init__(self) -> None:
         super().__init__(properties=Characteristic.NOTIFY)
 
-    def bind(self, service):
+    def bind(self, service: "CyclingSpeedAndCadenceService") -> _bleio.PacketBuffer:
```

Comment:
`CyclingSpeedAndCadenceService` is what is expected, yes.

Reply:
Yes, I think so, that is what is expected. Then it gets passed up to the superclass, which is less picky.